rwatch/ui/layer/action_bar_layer.c

```diff
@@ -102,7 +103,7 @@ static void draw(Layer *layer, GContext *context)
 #ifdef PBL_RECT
     graphics_fill_rect(context, full_bounds, 0, GCornerNone);
 #else
-    n_graphics_fill_circle(context, GPoint(full_bounds.origin.x + DISPLAY_COLS + 7, full_bounds.origin.y + (DISPLAY_COLS / 2)), DISPLAY_COLS + 20);
+    graphics_fill_circle(context, GPoint(full_bounds.origin.x + DISPLAY_COLS + 7, full_bounds.origin.y + (DISPLAY_COLS / 2)), DISPLAY_COLS + 20);
```

Comment:
did you check what happens if `full_bounds.origin.x != 0`? I feel like the addition of full_bounds is used to mimic the functionality of `graphics_fill_circle` vs `n_graphics_fill_circle`